src/init.cpp

```diff
@@ -399,10 +417,13 @@ int initApp(char const * const title, int fullscreen)
 	}
 
 	// create player classes
+	// TODO/FIXME: why isn't this in initGame? why is it in init.cpp?
```

Comment:
This was to not crash on frames checking for the mouse in the loading screen (caught this in debug)

Reply:
I see, this PR changes the UI code so it reads mousex/mousey etc variables directly when EDITOR is defined (so it doesn't crash). ergo this can probably go in initGame() now

Reply:
I think it was crashing the main game, not editor